src/gatling/scala/simulations/publication/CreatePublicationCivilAndFamilySimulation.scala

```diff
@@ -0,0 +1,40 @@
+package simulations.publication
+
+import io.gatling.core.Predef._
+import requests.publication.PublicationRequests.httpProtocol
+import scenarios.publication.PublicationScenarios
+import utils.auth.OAuthAPI.config
+import utils.auth.OAuthAPI.config.{onceUsers, rampUpUsers, rampUpUsersDuration}
+
+import scala.concurrent.duration.DurationInt
+import scala.language.postfixOps
+
+class CreatePublicationCivilAndFamilySimulation extends Simulation {
+
+  print("api url is" + config.dataManagementApi.url)
+  private val createPublicationCivilFamilyExec = PublicationScenarios.CreatePublicationCivilAndFamilyScenario
+    .inject(
+      atOnceUsers(onceUsers),
+      rampUsers(rampUpUsers) during(rampUpUsersDuration seconds)
+    )
+  // Per-06
```

Comment:
The // Per-06 comments I think can be removed. Similar in the other files below

Reply:
Done :)